desktop-widgets/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ include_directories(.
 	${CMAKE_BINARY_DIR}
 )
 
+set(CMAKE_AUTOMOC ON)
```

Comment:
Is this meant to be part of this pull request? It is not mentioned in the description.

Reply:
Thank you for reviewing the pull request.

The change is mentioned in the corresponding commit message. I needed this change to be able to include the manual in the desktop application.  The built-in documentation needs qt5-webkit which is outdated and not recommended due to security reasons (https://blogs.gnome.org/mcatanzaro/2022/11/04/stop-using-qtwebkit/).
Since it is only used to show the manual and not to browse the web, it may still be ok to use it.
That is why I fixed the CMake file. The change is not directly related to the user manual, why I put it in a separate commit.

I can mention the change in the pull request description or remove the commit that changes the cmake file. Maybe do a separate pull request?
What do you prefer?

Reply:
No, all good, thank you for your explanation.
But it looks like the DCO checker is not happy because the signed-off line does not match your commit author settings: https://github.com/subsurface/subsurface/pull/4054/checks?check_run_id=20682604486

Reply:
Thank you for the approval.
I changed my primary email address to match the Signed-off-by section in the commit message (instead of changing the commit message to the obscure email given by github). How to rerun the checks?

Reply:
Umm, no. QtWebkit is required for printing.
Sadly, and I wish we had found a different way to do this, but that's it's main purpose.
Showing the user manual with is i6s just easier because it's already there. That one could easily be replaced with something else.

I haven't done a full review, please give me time to actually wake up, deal with real life, and then make sure this doesn't break things

Reply:
Ok, managed to look at the change in more detail on my phone.

This should be ok. I got worried by your explanation 😏